test/models/user.js

```diff
@@ -30,11 +47,17 @@ User.init({
     defaultValue: 1,
   }
 }, {}, {
-  isValid: {
-    get() {
-      return this.status !== 1;
+  get isValid() {
+    return this.status !== 1;
+  },
+  set nickname(value) {
+    if (value === 'Zeus') {
+      this.attribute('nickname', 'V');
+    } else {
+      const { formatter: format } = this.constructor;
```

Comment:
format is a verb, which makes `format.formatName` a bit weird to read. Hence I'd prefer to leave it as `formatter` here.